src/Symfony/Component/HttpClient/HttpClient.php

```diff
@@ -61,6 +61,8 @@ public static function create(array $defaultOptions = [], int $maxHostConnection
             return new AmpHttpClient($defaultOptions, null, $maxHostConnections, $maxPendingPushes);
         }
 
+        @trigger_error('Install the curl extension or run "composer require amphp/http-client" for a performant HTTP client supporting HTTP/2', E_USER_NOTICE);
```

Comment:
I would suggest having a separate message when curl is installed, e.g.:
`@trigger_error((\extension_loaded('curl') ? 'Upgrade' : 'Install').' the curl extension or run "composer require amphp/http-client" to perform async HTTP operations, including full HTTP/2 support.', E_USER_NOTICE);`

Reply:
Updated.